src/Lib.hs

```diff
@@ -8,9 +8,10 @@ module Lib (
     evaluate,
     symbolRef, symbolList, constant, boolean, text, --void,
     defaultEnvironment,
-    oppositeOperator, addOperator, substractOperator, multiplyOperator, divideOperator,
+    oppositeOperator, addOperator, subtractOperator, multiplyOperator, divideOperator,
     notOperator, eqOperator, neqOperator, inferiorOperator,
     ifOperator, callOperator, defineOperator, lambdaOperator) where
+import Data.Functor ((<&>))
```

Comment:
Formatting bikeshedding, but I'd add an empty line above the import.

Reply:
Obviously, my bad for not adding one myself